README.md

```diff
@@ -1,91 +1,135 @@
-9. Quadtree-based point-in-polygon spatial join
-10. Quadtree-based point-to-linestring nearest neighbor distance
-11. Distance computation (point-point, point-linestring, linestring-linestring)
-12. Finding nearest points between point and linestring
+## Overview
+Now based on the [GeoArrow](https://github.com/geoarrow/geoarrow) format, cuSpatial accelerates vector geospatial operations through GPU parallelization. As part of the RAPIDS libraries, it's inherently connected to [cuDF](https://github.com/rapidsai/cudf), [cuML](https://github.com/rapidsai/cuml), and [cuGraph](https://github.com/rapidsai/cugraph), enabling GPU acceleration across entire workflows. 
 
-Future support is planned for the following operations:
+cuSpatial's Python API is closely matched to GeoPandas and data can seamlessly transition between the two:
+```python
+import geopandas
+from shapely.geometry import Polygon
+import cuspatial
 
-1. Temporal window query
-2. Temporal point query (year+month+day+hour+minute+second+millisecond)
-3. Grid-based indexing for points and polygons
-4. R-Tree-based indexing for Polygons/Polylines
+p1 = Polygon([(0, 0), (1, 0), (1, 1)])
+p2 = Polygon([(0, 0), (1, 0), (1, 1), (0, 1)])
+geoseries = geopandas.GeoSeries([p1, p2])
 
-## Install from Conda
-
-To install via conda:
-
-```shell
-conda install -c conda-forge -c rapidsai-nightly cuspatial
+cuspatial_geoseries = cuspatial.from_geopandas(geoseries)
+print(cuspatial_geoseries)
+```
+Output:
+```
+0    POLYGON ((0 0, 1 0, 1 1, 0 0))
+1    POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))
 ```
 
-## Install from Source
-
-To build and install cuSpatial from source:
-
-### Pre-requisite
-
-- gcc >= 7.5
-- cmake >= 3.23
-- miniconda
+For additional examples, browse our complete [API documentation](https://docs.rapids.ai/api/cuspatial/stable/), or check out our more detailed [notebooks](https://github.com/rapidsai/notebooks-contrib) - the NYC Taxi and Weather notebooks make use of cuSpatial.
+
+## Supported Geospatial Operations
+
+cuSpatial is constantly working on new features! Check out the [epics](https://github.com/orgs/rapidsai/projects/41/views/4) for a high-level view of our development, or the [roadmap](https://github.com/orgs/rapidsai/projects/41/views/5) for the details!
+
+### Core Spatial Functions
+- DE9-IM Functionality [Follow Along!](https://github.com/rapidsai/cuspatial/milestone/5)
+  - [Contains Properly](https://docs.rapids.ai/api/cuspatial/stable/api_docs/geopandas_compatibility.html#cuspatial.GeoSeries.contains_properly)
+- ST_Distance equivalent functionality [Follow Along!](https://github.com/rapidsai/cuspatial/issues/767)
+  - [Pairwise Linestring Distance](https://docs.rapids.ai/api/cuspatial/stable/user_guide/cuspatial_api_examples.html#cuspatial.pairwise_linestring_distance)
+  - [Pairwise Point-Linestring Distance](https://docs.rapids.ai/api/cuspatial/stable/api_docs/spatial.html#cuspatial.pairwise_point_linestring_distance)
+- Linestring-linestring intersection
```

Comment:
Linestring-linestring intersection should go under "Spatial relationship queries"?

Reply:
Yeah I think that's a good change, it's certainly a subset of them.